serving/samples/jib-app-java/README.md

```diff
@@ -0,0 +1,99 @@
+# Jib Sample App
+
+A sample app that demonstrates using [Jib](https://github.com/GoogleContainerTools/jib) to build Java applications
```

Comment:
Can you run this file through prettier.io with `--prose-wrap=always`? It will fold all the lines consistently.

Reply:
done